backend/stakepoold/server.go

```diff
@@ -310,23 +307,17 @@ func runMain(ctx context.Context) error {
 		log.Infof("block %v hash %v came in during GetTickets, refreshing...",
 			afterHeight, afterHash)
 	}
-
-	if err = nodeConn.NotifyBlocks(); err != nil {
-		fmt.Printf("Failed to register daemon RPC client for "+
-			"block notifications: %s\n", err.Error())
-		return err
-	}
```

Comment:
We were registering for blocks notifications but ignoring the notification. Is there a reason for this that I am not seeing? I think this will be used in the future though.

Reply:
Interesting observation. It was added in #178, I'm not sure why. Perhaps @chappjc remembers?